### ISSUE TITLE
Keeps asking for an update / Unable to uninstall

### ISSUE THREAD

**Issue by lorenzosfarra** [adelf/idea-php-dotenv-plugin#71]
Hi,

everytime I start my PhPStorm (2020.3.2) on Mac OS I have _Plugin Updates_ view with this plugin.
The required update is 2020.3 -> 2021.1.
I tried to perform the update multiple times, no error reported. Anyway the dialog appears everytime I open PhPStorm, always asking the 2020.3 -> 2021.1 update.
I wanted also to try to uninstall and install again the plugin: the "Uninstall" seems to work just fine, but the plugin is not removed.

How can I proceed? We can even try to totally uninstall and clear the plugin somehow manually, if you can provide the correct way to do it.
It doesn't happen with other plugins, so it's kind of related just to this one.

Thanks in advance for your support.
Lorenzo

**Comment by adelf** [comment]
Hi. I'm pretty sure that it is something with your PhpStorm installation... 
Could you repeat this try to install which should fail, then restart the PhpStorm, and then the main menu > Help >  Show Log in Explorer. The log file can contain important information in the last records... There are timestamps, so you can easily fetch all records after trying to install the plugin. They can help me to understand what's wrong.

**Comment by MaralisDaniel** [comment]
Have the same issue. But on Win10
Solved it by launching PhpStorm with administrative rights - it seems some files should be write in protected folder and there is no request to permit it

**Comment by lorenzosfarra** [comment]
@adelf I don't have a "Show Log" entry, so I looked for "Show Log" and "Log" in the help menu.
The results are:

- Tool Windows -> Event Log
- V8 Profiling -> Analyse V8 Profiling Log
- Composer -> Show Log
- Show Git Log

The only one that makes sense to me is Tool Windows -> Event Log but is empty.

-----

I had a look at the log file `$HOME/Library/Logs/JetBrains/PhpStorm2020.3/idea.log` that hopefully is what can give you the needed info, and I have found an interesting:

`2021-03-01 10:10:49,949 [  83253]   INFO - rketplacePluginDownloadService - File '/Users/lorenzo/Library/Caches/JetBrains/PhpStorm2020.3/plugins/idea-php-dotenv-plugin-2020.3.jar.zip' not found`

So I had a look (with `tail -f `) to catch the exact moment when the error occurs, and it happens exactly when I click on the "Update" button.
The UI does not give any feedback in this sense: the button just says "Installed". If I go back to the "Check for updates" dialog, the button says "Restart IDE".

Thanks,
  Lorenzo
